.github/workflows/build-develop.yml

```diff
@@ -0,0 +1,55 @@
+name: Build Develop
+
+permissions:
+  contents: read
+
+on:
+  push:
+    branches:
+      - 'develop'
+
+jobs:
+  run-eslint-and-test:
+    name: ESLint and Test
+    if: ${{ github.repository == 'RocketChat/Rocket.Chat.ReactNative' }}
+    uses: ./.github/workflows/eslint.yml
+
+  android-build-experimental-store:
+    name: Build Android Experimental
+    if: ${{ github.repository == 'RocketChat/Rocket.Chat.ReactNative' }}
+    uses: ./.github/workflows/build-android.yml
+    needs: [run-eslint-and-test]
+    secrets: inherit
+    with:
+      type: experimental
+      trigger: push
+
+  android-build-official-store:
+    name: Build Android Official
+    if: ${{ github.repository == 'RocketChat/Rocket.Chat.ReactNative' }}
+    uses: ./.github/workflows/build-official-android.yml
+    needs: [run-eslint-and-test]
+    secrets: inherit
+    with:
+      type: official
+      trigger: push
```

Comment:
If I got it right, you split build into two files: `build-pr` and `build-develop`.
A new trigger param was created that supports `pr` or `push`?
Can you make it `pr` or `develop` instead?
I think it's more clear.